src/azure-cli/azure/cli/command_modules/acr/_utils.py

```diff
@@ -25,7 +25,10 @@
     get_valid_variant,
     ACR_NULL_CONTEXT
 )
-from ._client_factory import cf_acr_registries
+from ._client_factory import (
+    cf_acr_registries,
+    cf_acr_tasks
```

Comment:
if only one method is using it you can also import it inside the method like this:
`from ._client_factory import cf_acr_tokens`